### PR TITLE
#128 quote consistency

### LINKED ISSUE #128: Consistent use of quotes
>Are we actually using single/double-quotes consistently in the documentation?

_Originally posted by @omikron-fp in https://github.com/FACT-Finder-Web-Components/documentation/pull/127_
@toarm 

We need to decide which types of quotes we want to use and make it uniform across the entire documentation.

### PR DESCRIPTION
**Pull request by damrath**
fixes #128

### CODE REVIEW

**Comment by omikron-fp** [comment]
Please rebase and _fast-forward_ merge it.

**Comment by wdutkiewicz-obj** [comment]
What about this pull request?

File: src/data/3.x/guides.js [hunk 1]
```diff
@@ -1,94 +1,94 @@
-// guides are the pages under the "Documentation" tab
+// guides are the pages under the Documentation" tab
```
Comment (wdutkiewicz-obj) [inline src/data/3.x/guides.js]: Opening quote was accidentally removed?

File: src/data/3.x/guides.js [hunk 2]
```diff
@@ -1,94 +1,94 @@
...
         },
         "your-first-search": {
-            path: "your-first-search",
-            title: "4. Your First Search",
+            path: `your-first-search`,
+            title: `4. Your First Search`,
         },
         "upgrade-guide": {
             path: "upgrade-guide",
             title: "5. Upgrade Guide 1.2.x to 3.0.0",
```
Comment (wdutkiewicz-obj) [inline src/data/3.x/guides.js]: Not changing " -> ` here?

**Comment by omikron-fp** [comment]
> What about this pull request?

I resolved the merge conflicts and took another look at the changes. Indeed only backticks. (and one typo)
Looks fine to me. Tested locally. Should be okay to merge.
Do you want to take another look before, @wdutkiewicz-obj?

Even if quotes are not 100% consistent after this PR, we shouldn't bother. Huge PR with little actual value.

**Comment by wdutkiewicz-obj** [comment]
> 
> 
> > What about this pull request?
> 
> I resolved the merge conflicts and took another look at the changes. Indeed only backticks. (and one typo)
> Looks fine to me. Tested locally. Should be okay to merge.
> Do you want to take another look before, @wdutkiewicz-obj?
> 
> Even if quotes are not 100% consistent after this PR, we shouldn't bother. Huge PR with little actual value.

I had a look and added some comments. a minor changes should be applied (should not take more than 5m).

File: src/data/3.x/guides.js (hunk 2 above)
Comment (omikron-fp) [reply src/data/3.x/guides.js]: Ah! That one didn't exist yet in the original PR 😄

**Pull request merged**